cfme/tests/networks/nuage/test_automation.py

```diff
@@ -12,3 +22,74 @@ def test_auth_attributes_of_nuage_ae_class(appliance):
     schema_fields = nuage_class.schema.schema_field_names
 
     assert all(expected in schema_fields for expected in expected_fields)
+
+
+@pytest.mark.parametrize(
+    'copy_ae_instance_to_new_domain',
+    ['ManageIQ/System/Event/EmsEvent/Nuage/nuage_enterprise_create'],
+    indirect=True)
+@pytest.mark.parametrize(
+    'ansible_repository',
+    [cfme_data.ansible_links.playbook_repositories.nuage],
```

Comment:
This breaks collection of tests when this entry isn't present in the cfme_data.yaml that is available.  Don't want to have a direct cfme_data reference here where it can cause an exception.

Consider parametrizing the playbook name (nuage), and have the fixture wrap the lookup in try/except to skip the test when that playbook repo name isn't available.

```
================ ERRORS ================
________ ERROR at setup of cfme/tests/networks/nuage/test_automation.py ___________
cfme/tests/networks/nuage/test_automation.py:33: in <module>
    [cfme_data.ansible_links.playbook_repositories.nuage],
.py2env/lib/python2.7/site-packages/lya/lya.py:68: in __getattr__
    if not (k.startswith('__') or k.startswith('_OrderedDict__')): return self[k]
E   KeyError: 'nuage'
```

Reply:
I changed how we pass the playbook into the fixture.